tests/10apidoc/01register.pl

```diff
@@ -82,11 +82,16 @@
 
 my $next_anon_uid = 1;
 
+sub create_localpart
```

Comment:
Tiny nit: I'd be tempted to call this `sprintf_localpart` or somesuch, to point out it just does some string formatting and doesn't e.g. talk to a Matrix server and Do Things.